apps/point-of-sale/vite.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig(({ mode }) => {
   const env = loadEnv(mode, process.cwd());
 
   return {
+    base: '/pos/',
```

Comment:
How do you want to deal with A/B testing like /pos-beta/ etc?

Reply:
Yeah, its not yet ready. This is still an issue I am trying to fix.

My idea is to rewrite the paths using nginx and just always have the project routed at "./" which kinda works? but it doesn't seem to actually run.